### PR TITLE
Get taxa populate spreadsheet

### PR DESCRIPTION
**Pull request by damianooldoni**
Progresses in merging checklists have been implemented in pipelines. In particular, this PR covers points 1-11 of Google  Document [Unified checklist - pipeline](https://docs.google.com/document/d/17TpZNcokL3NW5s9hkRAgvS6hyg6OhyfSFQEZ9xf7-No`)
Please, review following this order:
1. `./src/get_taxa.Rmd`
2. `./src/verify_synonyms.Rmd`
3. `./src/unify_info_from_checklists.Rmd`

### CODE REVIEW

File: src/get_taxa.Rmd [hunk 1]
```diff
@@ -189,10 +208,102 @@ Total number of taxa with valid distributions:
...
+taxon_info <- c("key", "scientificName", "species", "genus", "family", "order",
+                "class", "phylum", "kingdom",
+                "rank", "speciesKey", "taxonomicStatus", 
+                "acceptedKey", "accepted", "issues")
+
+taxon_backbone_info <- taxon_backbone_info %>%
+  select(one_of(taxon_info))
+```
+
+In case some information misses, add empty column(s):
```
Comment (peterdesmet) [inline src/get_taxa.Rmd]: @damianooldoni why is this step necessary?

File: src/get_taxa.Rmd [hunk 2]
```diff
@@ -189,10 +208,102 @@ Total number of taxa with valid distributions:
...
+taxon_backbone_info <- map(cols_to_add, 
+                           function(x) taxon_backbone_info %>% 
+                             mutate(!! sym(x) := NA)) %>% 
+  reduce(full_join, .init = taxon_backbone_info)
+```
+
+Next, we unlist column `issues` and we separate them by comma (if multiple issues are present):
+
+```{r unlist_issues}
+taxon_backbone_info <- taxon_backbone_info %>%
```
Comment (peterdesmet) [inline src/get_taxa.Rmd]: Is'nt there an easier way to do this?

File: src/get_taxa.Rmd (hunk 1 above)
Comment (damianooldoni) [reply src/get_taxa.Rmd]: Do you refer to the step of adding empty columns?

Comment (peterdesmet) [reply src/get_taxa.Rmd]: Indeed, I've deleted the step.

File: src/get_taxa.Rmd (hunk 2 above)
Comment (peterdesmet) [reply src/get_taxa.Rmd]: Apparently not, though I did use `na_if()`

**Comment by peterdesmet** [comment]
This PR has been reviewed in separate PR (#15). I will merge this one to master so we can continue with a clean slate and remove stale branches.

**Pull request merged**